terraform/docdb/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.4.3"
+    }
+  }
+}
+
+locals {
+  name_prefix     = replace("${var.environment}-${var.app_name}-${var.mongo_name}", "_", "-")
+  master_password = data.aws_secretsmanager_secret_version.master_password.secret_string
+}
+
+data "aws_secretsmanager_secret" "master_password" {
+  arn = aws_secretsmanager_secret.master_password.arn
+}
+
+data "aws_secretsmanager_secret_version" "master_password" {
```

Comment:
Do we need the data when we already have the resource?

Reply:
I'm not sure. What do you suggest we do?